src/mscorlib/src/System/Runtime/RuntimeImports.cs

```diff
@@ -28,5 +28,27 @@ internal unsafe static void RhZeroMemory(ref byte b, nuint byteLength)
 
         [DllImport(JitHelpers.QCall, CharSet = CharSet.Unicode)]
         extern private unsafe static void RhZeroMemory(byte* b, nuint byteLength);
+
+        // Non-inlinable wrapper around the QCall that avoids poluting the fast path
+        // with P/Invoke prolog/epilog.
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        internal unsafe static bool RhCopyMemoryWithReferences<T>(ref T destination, ref T source, int elementCount)
```

Comment:
It is called RhBulkMoveWithWriteBarrier in CoreRT - it maybe nice to call it the same.